From 8bba089ab7174b6cb06cb81903eb79c117aad9f8 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 27 Jun 2016 17:06:22 +0300
Subject: [PATCH 16/24] ARM: i.MX6: dts: clean up unnecesary code

Remove:
1) unnecessary GPRx assignment ( legacy from the 3.10.17_ga )
2) an incorrect analog audio routing

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>

Conflicts:
	arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
---
 arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi b/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
index db89991..33db95c 100644
--- a/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-cm-fx6.dtsi
@@ -13,9 +13,6 @@
 #include <dt-bindings/sound/fsl-imx-audmux.h>
 
 #define MX6QDL_GPR1	0x04 0x04 0x000 0x0 0x0
-#define MX6QDL_GPR6	0x18 0x18 0x000 0x0 0x0
-#define MX6QDL_GPR7	0x1c 0x1c 0x000 0x0 0x0
-#define MX6QDL_GPR1    0x04 0x04 0x000 0x0 0x0
 
 / {
 	memory {
@@ -107,8 +104,6 @@
 		simple-audio-card,routing =
 			"Headphone Jack", "RHPOUT",
 			"Headphone Jack", "LHPOUT",
-			"LLINEIN", "Line Jack",
-			"RLINEIN", "Line Jack",
 			"MICIN", "Mic Bias",
 			"Mic Bias", "Mic Jack";
 		simple-audio-card,format = "i2s";
@@ -219,9 +214,6 @@
 		pinctrl_hog: hoggrp {
 			fsl,pins = <
 				MX6QDL_GPR1 0x48400005
-				/* ipu3 QoS */
-				MX6QDL_GPR6 0x007f007f
-				MX6QDL_GPR7 0x007f007f
 				/* SATA PWR */
 				MX6QDL_PAD_ENET_TX_EN__GPIO1_IO28 0x80000000
 				MX6QDL_PAD_EIM_A22__GPIO2_IO16 0x80000000
-- 
1.9.1

